app/assets/stylesheets/global.scss

```diff
@@ -0,0 +1,7 @@
+.top-bar .plain-text {
+  font-family: "Helvetica Neue", Helvetica, Roboto, Arial, Sans-Serif;
+  font-size: 13px;
+  line-height: 45px;
+  padding: 0 15px;
+  color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.